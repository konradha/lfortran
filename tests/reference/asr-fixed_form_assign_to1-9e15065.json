{
    "basename": "asr-fixed_form_assign_to1-9e15065",
    "cmd": "lfortran --indent --fixed-form --show-asr --no-color {infile} -o {outfile}",
    "infile": "tests/fixed_form_assign_to1.f",
    "infile_hash": "d76077afdd3f3c22b5e6769f2365e819c0fe06298487bd65c6e59526",
    "outfile": null,
    "outfile_hash": null,
    "stdout": "asr-fixed_form_assign_to1-9e15065.stdout",
    "stdout_hash": "fb6a4f9a91fc56f3fde0452d68298d283db1fe6b106955f31f10ad3a",
    "stderr": null,
    "stderr_hash": null,
    "returncode": 0
}