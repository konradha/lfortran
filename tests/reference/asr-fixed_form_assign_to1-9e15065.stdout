(TranslationUnit 
    (SymbolTable
        1
        {
            assign3:
                (Program 
                    (SymbolTable
                        2
                        {
                            next:
                                (Variable 
                                    2 
                                    next 
                                    Local 
                                    () 
                                    () 
                                    Default 
                                    (Integer 8 []) 
                                    Source 
                                    Public 
                                    Optional 
                                    .false.
                                )
                            
                        }) 
                    assign3 
                    [] 
                    [(GoToTarget 
                        10
                    )
                    (= 
                        (Var 2 next) 
                        (IntegerConstant 10 (Integer 8 [])) 
                        ()
                    )]
                )
            
        }) 
    []
)
